### PR TITLE
Added Animated Colourful Background

### LINKED ISSUE #341: [Feature] New Animated colorful background
### Description

I will add a new **Animated colorful background** in backgrounds component.
For this i will use HTML and CSS.

Demo video of this is attached.

Kindly assign this to me under **IWOC024**

### Screenshots

https://github.com/Rakesh9100/Beautiify/assets/149922307/5b154ca7-e3a4-4f30-8b82-19b81d29e2ad

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

### PR DESCRIPTION
**Pull request by Prince-kushwaha-2004**
# Fixes Issue🛠️

#341 New Animated colorful background

Closes #341 

# Description👨‍💻 

Added new **Animated colourful background** in background component.
For this I had used **HTML** and **CSS.**

# Type of change📄

- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅

This has been tested by me by running the website, no error found!!

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷

https://github.com/Rakesh9100/Beautiify/assets/149922307/47f4d8d8-5368-4498-8a84-bf58d630ea2f

### CODE REVIEW

File: assets/html_files/backgrounds.html [hunk 1]
```diff
@@ -42,6 +42,21 @@
     <section>
         <div class="heading">Backgrounds</div>
         <div class="contain">
+            <div class="box">
+                <h1>Animated Colouful Background</h1>
```
Comment (Rakesh9100) [inline assets/html_files/backgrounds.html:46]: Pls correct the spelling @Prince-kushwaha-2004

Comment (Prince-kushwaha-2004) [reply assets/html_files/backgrounds.html]: done👍

**Comment by Prince-kushwaha-2004** [comment]
@Rakesh9100 
kindly check now.

**Comment by Rakesh9100** [comment]
Merged ✨ @Prince-kushwaha-2004

**Pull request merged**